### PR TITLE
chore: add v4.0.0 changelog

### PR DESCRIPTION
**Pull request by mdlufy**
Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,127 @@ All notable changes to this project will be documented in this file. See
...
+  - `CardMedium`, `CardLarge` add new components ([#7260](https://github.com/taiga-family/taiga-ui/pull/7260)).
+  - `Cell` add new component ([#5460](https://github.com/taiga-family/taiga-ui/pull/5460),
+    [#7235](https://github.com/taiga-family/taiga-ui/pull/7235)).
+  - `Navigation` implement grid (https://github.com/taiga-family/taiga-ui/pull/8377). Simplify 2nd level aside
+    (https://github.com/taiga-family/taiga-ui/pull/8141).
+- **@taiga-ui/addon-commerce:**
+  - `InputCard` migrate to new `Textfield` ([#7581](https://github.com/taiga-family/taiga-ui/pull/7581)).
+- **@taiga-ui/addon-doc:**
+  - `ThemeSwitcher` add new component (https://github.com/taiga-family/taiga-ui/pull/7966).
+  - `Markdown` add pipe (https://github.com/taiga-family/taiga-ui/pull/6294).
```
Comment (splincode) [inline CHANGELOG.md:93]: image

Comment (mdlufy) [reply CHANGELOG.md:93]: probably using `markdown-it` utils instead TuiMarkdownPipe?
it is worth replacing this point with something else or removing it

Comment (splincode) [reply CHANGELOG.md:93]: just remove it

Comment (mdlufy) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,126 @@ All notable changes to this project will be documented in this file. See
...
+  - `Textfield` add new component ([#6298](https://github.com/taiga-family/taiga-ui/pull/6298)).
+  - `Icon` add new component ([#5872](https://github.com/taiga-family/taiga-ui/pull/5872)). `Icons` add new directive
+    ([#6053](https://github.com/taiga-family/taiga-ui/pull/6053)) `Icon` add new pipe
+    (https://github.com/taiga-family/taiga-ui/pull/7244).
+  - `Icons` new approach **based on mask** ([#7752](https://github.com/taiga-family/taiga-ui/pull/7752),
+    [#7658](https://github.com/taiga-family/taiga-ui/pull/7658),
+    [#7714](https://github.com/taiga-family/taiga-ui/pull/7714)).
+  - `Link` uses `Icon` instead of legacy `Svg` ([#7056](https://github.com/taiga-family/taiga-ui/pull/7056)). `Link`
+    refactor to use `Icons` directive ([#7170](https://github.com/taiga-family/taiga-ui/pull/7170)).
+  - `Flag` add new pipe ([#3725](https://github.com/taiga-family/taiga-ui/pull/3725)).
```
Comment (nsbarsukov) [inline CHANGELOG.md:24]: It already exists in `v3`:
https://taiga-ui.dev/v3/pipes/flag

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,126 @@ All notable changes to this project will be documented in this file. See
...
+  - `Scrollbar` add mode (https://github.com/taiga-family/taiga-ui/pull/8246).
+  - `Notification` simplify to directive (https://github.com/taiga-family/taiga-ui/pull/8168). Add interactive mode
+    (https://github.com/taiga-family/taiga-ui/pull/6760).
+  - `HintOverflow` add new directive (https://github.com/taiga-family/taiga-ui/pull/8111).
+  - `DropdownContext` implement for `IOS` (https://github.com/taiga-family/taiga-ui/pull/7814).
+  - `Root` use `Platform` directive (https://github.com/taiga-family/taiga-ui/pull/7931).
+  - `Tooltip` add new directive (https://github.com/taiga-family/taiga-ui/pull/7810).
```
Comment (nsbarsukov) [inline CHANGELOG.md:31]: `Tooltip` already exists in `v3`.
https://taiga-ui.dev/v3/components/tooltip

Probably, we should change this line a little bit.
Something like: "`Tooltip` are refactored from component to just directive"

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,126 @@ All notable changes to this project will be documented in this file. See
...
+  - `Segmented` add new component ([#6527](https://github.com/taiga-family/taiga-ui/pull/6527),
+    [#7192](https://github.com/taiga-family/taiga-ui/pull/7192)). Tabs refactor and drop `MobileTabs` in favor of
+    `Segmented` ([#7047](https://github.com/taiga-family/taiga-ui/pull/7047)).
+  - `Fade` add new directive ([#5169](https://github.com/taiga-family/taiga-ui/pull/5169)).
+  - `Tabs` use `Icons` ([#7177](https://github.com/taiga-family/taiga-ui/pull/7177)).
+  - `Avatar` add colored icons mode ([#7806](https://github.com/taiga-family/taiga-ui/pull/7806)).
+  - `Connected` add directive for `Cell` and `Stepper` ([#8025](https://github.com/taiga-family/taiga-ui/pull/8025)).
+  - Use `@maskito/phone` ([#7580](https://github.com/taiga-family/taiga-ui/pull/7580)), use signals
+    ([#7662](https://github.com/taiga-family/taiga-ui/pull/7662)), use host directives
+    ([#8052](https://github.com/taiga-family/taiga-ui/pull/8052)) in `InputPhoneInternational`.
```
Comment (nsbarsukov) [inline CHANGELOG.md:58]:
```suggestion
  - Use `@maskito/phone` ([#7580](https://github.com/taiga-family/taiga-ui/pull/7580)) in `InputPhoneInternational`.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,126 @@ All notable changes to this project will be documented in this file. See
...
+  - `DropdownMobile` add new directive ([#7428](https://github.com/taiga-family/taiga-ui/pull/7428),
+    [#8094](https://github.com/taiga-family/taiga-ui/pull/8094)).
+- **@taiga-ui/addon-table:**
+  - `Reorder` add possibility to transfer a template (https://github.com/taiga-family/taiga-ui/pull/6967).
+  - `TableSort` support nullable values (https://github.com/taiga-family/taiga-ui/pull/7888).
+- **@taiga-ui/i18n**:
+  - `Hebrew` add support (https://github.com/taiga-family/taiga-ui/pull/8242).
+  - `Belarusian` add support (https://github.com/taiga-family/taiga-ui/pull/6421).
+
+Changelog may be updated in the near future.
```
Comment (nsbarsukov) [inline CHANGELOG.md:106]: Remove this line

**Review by nsbarsukov — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (mdlufy) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 3 above)
Comment (mdlufy) [reply CHANGELOG.md]: changed

File: CHANGELOG.md (hunk 4 above)
Comment (mdlufy) [reply CHANGELOG.md]: updated

File: CHANGELOG.md (hunk 5 above)
Comment (mdlufy) [reply CHANGELOG.md]: removed

**Review by waterplea — APPROVED** [review]

**Pull request merged**